### ISSUE TITLE
UPgrade to FTD 6.4+

### ISSUE THREAD

**Issue by joseanda** [pcardotatgit/FDM_Managed_FTD_6.3_Device_Quick_Start#1]
Hi Are you planning to upgrade for FTD 6.4+?

**Comment by pcardotatgit** [comment]
Hi Joseande.   

It is already done. I created a new repo for this :  https://github.com/pcardotatgit/FDM_Add_Security_Rules.

6.3  to 6.6.

6.7 update are on my todo list

Regards

Patrick

**Comment by joseanda** [comment]
Thanks Patrick. I am struggling building an script to add network object
groups. Do you have one?

**Comment by pcardotatgit** [comment]
Yes ,  I the repo I mentioned to you,  have a look to the script named : 

3_fdm_add_network_object_group.py 

You must first create single objects

Patrick
